Dockerfile

```diff
@@ -45,7 +34,22 @@ ENV LC_ALL C.UTF-8
 
 RUN \
   yum update --security -y && \
-  if [ "$JDIST" = 'jdk' ]; then JAVA_PKG_TYPE="devel"; else JAVA_PKG_TYPE="headless"; fi && \
+  JAVA_PKG_TYPE="headless" && \
```

Comment:
just replace that variable with it's value. These images should not install anything but headless